packages/wrangler/src/index.tsx

```diff
@@ -139,14 +139,14 @@ function demandOneOfOption(...options: string[]) {
 }
 
 export async function main(argv: string[]): Promise<void> {
-  const yargs = makeCLI(hideBin(process.argv))
+  const wrangler = makeCLI(hideBin(process.argv))
```

Comment:
I like this change. We could then go a bit further perhaps and let `makeCLI` just be `yargs`?
When I first came across this `makeCLI()` function, I thought it must be something we had defined; and only later realised that this is just a custom name we have given to the yargs library namespace.

Reply:
I think that's a good idea, but I'll have to change all the `yargs` references below, and I'll save that for a future PR.